qemu/tests/x86_cpu_model.py

```diff
@@ -6,20 +6,11 @@
 from virttest import error_context, utils_misc, env_process
 
 
-@error_context.context_aware
-def run(test, params, env):
+def check_supported_model(params, test):
```

Comment:
Remind me why you need add this function, where else will you use it?

Reply:
As the describe said : Check if current cpu model is in the supported lists.
Don't we need to check cpu_model at first?